app/views/content/life-as-a-teacher/teaching-as-a-career/what-its-like-to-be-a-teacher/_article.html.erb

```diff
@@ -0,0 +1,44 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+<h2 class="heading--box-blue"> An example day in the life of a teacher</h2>
+ <p>Your school day could vary hugely depending on the age group or subject you teach, the number of pupils in your class and many other factors. However, some things, like the timetable of the day, are more predictable. This example day is designed to give you an idea of the sorts of things you could be doing on an average day.</p>
+ <h3>Morning</h3>
+  <h4>7:30am arrive at school</h4>
+   <p>Teachers often start their day early, arriving at school somewhere between 7:00am to 8:00am. The morning is usually spent preparing resources for the day ahead so that everything is organised for when the pupils arrive.</p>
+  <h4> 8:45am pupils arrive</h4>
+   <p>Depending on your role, you might be responsible for a form group or registering pupils. During this time, pupils typically engage in morning activities such as word puzzles. It's also an opportunity to talk to students individually to check their wellbeing.</p>
+    <%= render 'content/shared/quotes/quote_secondary_phil' %>
```

Comment:
These quotes might be better in the smaller size, to try to make the page shorter.